kyuubi-plugin/src/main/java/org/apache/kyuubi/plugin/SessionConfAdvisor.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.plugin;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/** Provide the session configuration according to the user and session configuration. */
+public interface SessionConfAdvisor {
```

Comment:
Since method `getConfOverlay` will overwrite the session conf, but method `getConfSuggestion` not, shall we add comments for these two methods?